zephyr/src/indigo_api_callback_dut.c

```diff
@@ -131,7 +131,7 @@ static int get_control_app_handler(struct packet_wrapper *req, struct packet_wra
 static int reset_device_handler(struct packet_wrapper *req, struct packet_wrapper *resp) {
     int status = TLV_VALUE_STATUS_NOT_OK;
     char *message = TLV_VALUE_RESET_NOT_OK;
-    char role[TLV_VALUE_SIZE], log_level[TLV_VALUE_SIZE], band[TLV_VALUE_SIZE];
+    char role[32], log_level[32], band[32];
```

Comment:
The moment you write `also` in the commit log that means the change can go in another commit.